### PR TITLE
[TS] LPS-143091 - Cards Dropdown is shown when no elements are passed

### PR DESCRIPTION
**Pull request by victorg1991**
This is a resend from: https://github.com/liferay-frontend/liferay-portal/pull/1714

> This fixes case https://issues.liferay.com/browse/LPS-143091
> 
> List options is showed when users have only view permissions over webcontent templates. Since they have no permissions, it is empty (see image attached)
> 
> Steps to reproduce:
> 
> 1. Create a new site 'editor' rol including only view permissions for web content structures and web content templates.
> 2. Add a new user, make him member of Liferay DXP site and assign him with the new role.
> 3. Add a new structure 'test' and template 'test' for that structure.
> 4. Log in with the new user.
> 5. Acces web content templates tab and configure the cards/icon view.
> 
> **Expected result**: user just sees 'test' with no dropdown menu available.
> 
> **Current result:** user sees 'test' and is able to click on the actions menu (three ellipsis button), and an empty list shows up.
> empty_list
> 
> It does not break https://issues.liferay.com/browse/LPS-136174

### CODE REVIEW

**Comment by victorg1991** [comment]
Can you take a look @carloslancha? 😇

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default function DropdownMenu({
 				className={classNames({
 					'dropdown-action': actionsDropdown,
 				})}
-				items={normalizeDropdownItems(items)}
+				items={normalizeDropdownItems(items) || []}
```
Comment (carloslancha) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js:43]: Why are we not using `useMemo` here following the same pattern?

**Review by carloslancha — COMMENTED** [review]
LGTM, just left a minor nit. Thx!

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js (hunk 1 above)
Comment (victorg1991) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js:43]: Dropdown doesn't have any external state so it won't rerender AFAIK

Comment (carloslancha) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js:43]: <a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F3oEjI5VtIhHvK37WYo%2Fgiphy.gif&token=nice" data-gitmeme-token="nice"></a>

**Comment by carloslancha** [comment]
ci:forward

**Pull request closed without merging**